src/isar/storage/blob_storage.py

```diff
@@ -40,13 +40,14 @@ def store(self, inspection: Inspection, metadata: MissionMetadata):
             inspection=inspection, metadata=metadata, filename=local_path.name
         )
 
-        self._upload_file(path=local_path, data=inspection.data)
+        inspection_path = self._upload_file(path=local_path, data=inspection.data)
         self._upload_file(path=local_metadata_path, data=metadata_bytes)
+        return inspection_path
 
-    def _upload_file(self, path: Path, data: bytes):
+    def _upload_file(self, path: Path, data: bytes) -> str:
         blob_client = self._get_blob_client(path)
         try:
-            blob_client.upload_blob(data=data)
+            inspection_path = blob_client.upload_blob(data=data)
```

Comment:
```suggestion
            blob_properties = blob_client.upload_blob(data=data)
```
As it is not an inspection path that is returned from `upload_blob`

Reply:
done